### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by matthiassb**
Includes:
* Sweetalert
* jQuery Backstretch
* Express-Session module
* New Routers
* Router Cleanup

image

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ var marked        = require('marked');
 var validator     = require('validator');
 var extend        = require('extend');
 var hogan         = require('hogan-express');
-var basic_auth    = require('basic-auth-connect');
+var session       = require('express-session');
```
Comment (ryanlelek) [inline app/index.js]: @gilbitron - Do you feel we should keep HTTP Basic as an alternative option and then default to Matthias' session auth? Or just remote HTTP Basic entirely?

**Comment by ryanlelek** [comment]
Comments added to both PRs and getting Gilbert's feedback
Then we'll get this merged and published

**Comment by matthiassb** [comment]
Sounds good!

**Comment by gilbitron** [comment]
Looks good. Defaulting to session auth seems the right way to go.

**Pull request merged**